.gitattributes

```diff
@@ -7,6 +7,23 @@
 src/Mod/Fem/*.h text
 src/Mod/Fem/*.cpp text
 src/Mod/Fem/*.csv text
+src/Mod/Fem/*.py text
+src/Mod/Fem/*.xml text
+src/Mod/Fem/*.txt text
+src/Mod/Fem/*.md text
+src/Mod/Fem/*.ui text
+src/Mod/Fem/*.qrc text
+src/Mod/Fem/*.bat text
+src/Mod/Fem/*.dat text
+src/Mod/Fem/*.ts text
```

Comment:
these are translation files. Since I am not sure  if this could cause trouble we should comment  it. Furthermore we should only use a file ending here if we are sure all files of this ending have 100 % unix line endings.

Reply:
Per my investigations, at this point (i.e. after commit ab2f02), all files in the Fem sub-directory structure have unix-style line-endings. By including these file extensions in the `.gitattributes` file, we ensure that these file remain as such moving forward.

Finally, based on my mime-type studies using the python script provided in my previous comment, all four of the file-extensions you list here are, in fact, simple text-files.

Reply:
great this is very important!

My comment was only for the *.ts files. The others are fine. I just have never ever done anything in the regard of the *.ts files. Thus I do not know if we could run intot trouble here.